OmiseSDK/PaymentCreatorController.swift

```diff
@@ -540,6 +540,7 @@ extension PaymentCreatorController {
         .installmentBAY,
         .installmentFirstChoice,
         .installmentBBL,
+        .installmentEzypay,
```

Comment:
Is EZPay not a Malaysian payment methods? So should it be added to `malaysiaDefaultAvailableSourceMethods` and not `thailandDefaultAvailableSourceMethods`?

Reply:
that is right. fixed now